backend/dissemination/migrations/0003_note_contains_chart_or_table.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 4.2.3 on 2023-09-15 05:50
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("dissemination", "0002_additionalein"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="note",
+            name="contains_chart_or_table",
+            field=models.TextField(
+                default="",
+                help_text="Census mapping: ",
```

Comment:
I'm not entirely sure what our best practices should be for adding new columns / assigning defaults.

We'll also want to regenerate this migration once the help_text is updated.

Reply:
This should not have a default, as it should be required by the schema, and there should always be a Y/N in this column. I think that would eliminate the need for a default?